osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -26,6 +27,8 @@ public abstract class WorkingBeatmap : IDisposable
 
         public PlayMode PlayMode => beatmap?.BeatmapInfo?.Mode > PlayMode.Osu ? beatmap.BeatmapInfo.Mode : PreferredPlayMode ?? PlayMode.Osu;
 
+        public readonly Bindable<Mod[]> Mods = new Bindable<Mod[]>();

```

Comment:
Can't someone just do `Mods.Value[0] = someNewMod` and there will be no callback?